src/components/FileInput/index.vue

```diff
@@ -186,14 +256,41 @@ export default Vue.extend({
       type: String,
       default: null,
     },
+    something: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
Don't understand the purpose of this prop. Please remove it everywhere, if it's unnecassary